tutorials/surrogate/cifar10_example.py

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/env python3
+
+import random
+import os
+import sys
+import enum
+from typing import Union, Dict, Tuple, Generator
+
+import torch
+from torch import nn
+from torch.utils.data import DataLoader, TensorDataset
+
+from torchvision.datasets import CIFAR10
+from torchvision.transforms import Compose, ToTensor, Normalize
+from torchmetrics import Accuracy
+
+from mpi4py import MPI
+
+from propulate import Islands
+from propulate.utils import get_default_propagator
+
+# from tutorials.surrogate.mock_surrogate import MockSurrogate
+# from tutorials.surrogate.log_surrogate import LogSurrogate
+from tutorials.surrogate.static_surrogate import StaticSurrogate
+# from tutorials.surrogate.dynamic_surrogate import DynamicSurrogate
```

Comment:
Can we do something to avoid commented-out lines in the code, e.g., just import all of them at once (unused imports depending on which surrogate is actually used) or something similar? I already made a comment about this in the other example, I guess.